### PR TITLE
Fix connection to postgres from grafana

### PR DESCRIPTION
**Pull request by dm3ch**
/close https://github.com/chainstack/graph-deployment/issues/58

### CODE REVIEW

File: helmfile/INFRA_README.md [hunk 1]
```diff
@@ -29,6 +29,21 @@ This includes Prometheus, Exporters, Grafana and basic kubernetes dashboards
 ### Graph Grafana dashboards
 Includes The Graph specific dashboards for grafana. Also includes configuration of postgres datasource for grafana.
 
+Some of dashboards was inspired or based on https://github.com/StakeSquid/graphprotocol-mainnet-docker/tree/advanced/grafana/provisioning/dashboards
```
Comment (CaJIbHuK) [inline helmfile/INFRA_README.md:32]:
```suggestion
Some of the dashboards were inspired or based on https://github.com/StakeSquid/graphprotocol-mainnet-docker/tree/advanced/grafana/provisioning/dashboards.
```

File: helmfile/INFRA_README.md [hunk 2]
```diff
@@ -29,6 +29,21 @@ This includes Prometheus, Exporters, Grafana and basic kubernetes dashboards
...
+
+**Postgres datasource**:
+Unfortunately postgresql password (used for postgres datasource) is not known before graph installation.
+In order to make dashboards that uses postgres data work you should pass postgres password to values after graph installation and reapply infra helmfile.
```
Comment (CaJIbHuK) [inline helmfile/INFRA_README.md:36]:
```suggestion
In order to make the dashboards that uses postgres data work you have to pass postgres password to values after graph installation and reapply infra helmfile.
```

File: helmfile/INFRA_README.md [hunk 3]
```diff
@@ -29,6 +29,21 @@ This includes Prometheus, Exporters, Grafana and basic kubernetes dashboards
...
+
+You can get needed postgres password using following command:
+```
+kubectl -n <graph-node namespace> get secret node.postgres-postgres.credentials.postgresql.acid.zalan.do -o json | jq -r ".data.password" | base64 -d`
+```
+
+You should pass it to `postgresDatasource.password` field in `infra.secret.yaml` and run appli command again.
```
Comment (CaJIbHuK) [inline helmfile/INFRA_README.md:43]:
```suggestion
You have to pass it to `postgresDatasource.password` field in `infra.secret.yaml` and run `apply` command again.
```

File: helmfile/INFRA_README.md [hunk 4]
```diff
@@ -29,6 +29,21 @@ This includes Prometheus, Exporters, Grafana and basic kubernetes dashboards
...
+
+After that you should restart grafana pod to make changes take effect.
```
Comment (CaJIbHuK) [inline helmfile/INFRA_README.md:45]:
```suggestion
After that, restart grafana pod to make changes take effect.
```

**Review by CaJIbHuK — CHANGES_REQUESTED** [review]
@akegaviar please review

**Review by akegaviar — APPROVED** [review]

**Review by CaJIbHuK — APPROVED** [review]

**Pull request merged**